salt/states/x509.py

```diff
@@ -578,7 +578,7 @@ def certificate_managed(
     for append_cert in append_certs:
         file_args["contents"] += __salt__["x509.get_pem_entry"](
             append_cert, pem_type="CERTIFICATE"
-        )
+        ).decode("utf-8")
```

Comment:
We're trying to avoid direct calls to `decode` and `encode` functions. Previous code appending `file_args["contents"]` uses `salt.utils.stringutils.to_str` for data in this field. I think it would be more correct to use the same utility function here.

Reply:
Alright, that should be pretty straight forward.